src/index.js

```diff
@@ -1,36 +1,116 @@
 import './style.css';
 
-const tasks = [
-  {
-    description: 'Wake up',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'Pray',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Eat breakfast',
-    completed: false,
-    index: 2,
-  },
-];
-
+let tasks = [];
 const taskWrapper = document.querySelector('.to-dos');
+const newTask = document.querySelector('.new-task');
+const addNewTask = document.querySelector('.submit');
+const clearAll = document.querySelector('.clear-all');
 
-const showList = () => {
-  taskWrapper.innerHTML = '';
+const addToLocalStorage = () => {
+  localStorage.setItem('myTasks', JSON.stringify(tasks));
+};
+
+const getFromLocalStorage = () => {
+  if (localStorage.getItem('myTasks')) {
+    tasks = JSON.parse(localStorage.getItem('myTasks'));
+  }
+  return tasks;
+};
+
+const resetIndex = (tasks) => {
```

Comment:
- [ ] Kindly make sure that by default, new tasks should have the property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5). For example, if you take a look at the below screenshot, you notice the difference 👇 
![Screenshot (199)](https://user-images.githubusercontent.com/67087664/155324274-499e8080-f033-4cca-906c-26cabe1a2495.png)